### ISSUE TITLE
How do optical flow and RGB fuse in TSN's code?

### ISSUE THREAD

**Issue by djy9809** [open-mmlab/mmaction2#1602]
I'm sorry to raise such a question. Since I just came into contact with this knowledge and didn't have a good command of coding, I didn't find the code about optical flow and RGB fusion in the process of reading TSN related code. 
Can you tell me where it fuses the RGB and optical flow branches in a 1:1 ratio?
Thank you very much!

**Comment by kennymckormick** [comment]
Yes, usually we use 1: 1, but you can also tune the fusing ratio to obtain a better result on ur dataset.

**Comment by djy9809** [comment]
> Yes, usually we use 1: 1, but you can also tune the fusing ratio to obtain a better result on ur dataset.

Thank you very much for your reply. 
I'm sorry, maybe I didn't describe my question clearly. 
What I want to know is where can I find this fusion ratio 1:1? 
Is it in test?

**Comment by kennymckormick** [comment]
The scripts for multi-modality fusion is https://github.com/open-mmlab/mmaction2/blob/master/tools/analysis/report_accuracy.py
